JustSaying.AwsTools/QueueCreation/AmazonQueueCreator.cs

```diff
@@ -6,27 +5,49 @@ namespace JustSaying.AwsTools.QueueCreation
 {
     public class AmazonQueueCreator : IVerifyAmazonQueues
     {
+        private readonly IAwsClientFactoryProxy awsClientFactory;
+        private readonly IRegionResourceCache<SqsQueueByName> queueCache = new RegionResourceCache<SqsQueueByName>();

```

Comment:
@payman81 , thanks, will update this. I've seen underscores being used before, it's just we use different Resharper settings.  Will ping once done.